main/src/resources/benchmark/Benchmark.Inliner.Optimized.flix

```diff
@@ -21,6 +21,15 @@ mod Benchmark.Inliner.Optimized {
         case z :: zs => mp(zs, z + 1 :: acc)
     }
 
+    pub def fmp(l: List[Int32], acc: List[Int32]): List[Int32] = match l {
```

Comment:
Not really sure why these functions could not have been given proper names? :)

Reply:
I thought it was easier to see that it was the (manually) lifted local defs

Reply:
Alright